src/main/resources/application.properties

```diff
@@ -1,8 +1,13 @@
-spring.application.name=spring.boot
-spring.datasource.url=jdbc:mysql://localhost:3306/test?serverTimezone=UTC
-spring.datasource.username=root
-spring.datasource.password=Cencei89
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-
-spring.jpa.hibernate.ddl-auto=create-drop
-spring.jpa.show-sql=true
+spring.datasource.url=jdbc:h2:mem:testdb
```

Comment:
why are you using in-memory db in your program? Let's use MySQL for these purposes. Feel free to use H2 for tests :)